rank/__tests__/eval.test.ts

```diff
@@ -17,6 +17,8 @@ beforeAll(async () => {
   searchTemplates = await getTemplates()
 })
 
+const queryEnv = process.env.RANK_QUERY_ENVIRONMENT || 'production'
```

Comment:
Not sure what @jamesgorrie's view on this might be, but to me the `queryEnv` and the `index` for testing feel like variables we should set as CLI arguments rather than (as well as? instead of?) environment variables.

Reply:
I'd be cool with that (and prefer to do so), I was just too lazy to look up how command-line argument parsing works in Jest. 😛 

Happy to add it if you think the core concept is solid.

Reply:
[yargs is your friend](https://www.npmjs.com/package/yargs)

Reply:
Agree with being a arg, probably not worth setting the default and error if it's not set.